integration/testrunner/grpc.go

```diff
@@ -145,14 +150,18 @@ func testGRPCEndpoint(ctx context.Context, t *testing.T, g *GRPC) {
 		} else {
 			t.Errorf("Did not get err as expected. Instead got reply: %v", reply)
 		}
-
-		query := g.makeQueryForGRPCUnary(id)
-		queryIfAuditLogExistsWithRetries(ctx, t, query, g.Config, "unary_fail")
+		query := g.makeQueryForGRPC(id)
+		t.Logf("querying BigQuery:\n%s", query.Q)
+		wantNum := 1
```

Comment:
I think it's fine to inline the `wantNum` in the func call

Reply:
done